.gitignore

```diff
@@ -4,4 +4,5 @@ dist/
 package-lock.json
 .rpt2_cache
 .vscode
-*.patch
\ No newline at end of file
+*.patch
+.history/
```

Comment:
Can you please add an empty line at the bottom of this file if you have a moment?